### ISSUE TITLE
FR: Reuse Volvo Search Block for Road Choice Blogs

### ISSUE THREAD

**Issue by ZoeBeykirch** [hlxsites/vg-roadchoice-com#239]
**Dependency:** https://github.com/hlxsites/vg-volvotrucks-us/issues/532

### Context/Description

As a Road Choice content author, I want to reuse the search block from the [Volvo press release](https://www.volvotrucks.us/news-and-stories/press-releases/) website so that I can bring the same search functionality to my blogs on the Road Choice website. 

### Acceptance Criteria

**User AC:**
- Users can perform keyword searches using the search block.
- Search results are relevant only to the Road Choice blog articles. 
- Users can filter by pre-defined category tags. 

**Non-functional AC:**
- The search block design matches the existing Road Choice website CSS design. 
- The search functionality matches the existing Volvo Press Release search and tag filter experience. 
- Search queries return results within 2 seconds. 

### Out of scope

- General search improvements on the site outside of the blog section.
- Predictive or autocomplete search functionality. 

### Attachment / link to Figma

The only difference would be to use the red Road Choice color and brand fonts: 

![Image](https://github.com/hlxsites/vg-roadchoice-com/assets/125895595/f13f409a-2870-48ad-940b-bf088d8fa4eb)

**Comment by mollyaljenkins** [comment]
This ticket will be on hold until the upgrade of VT search and filtering as the current functionality is not compatible.

**Comment by BeckyMedlin** [comment]
Blocked by https://github.com/hlxsites/vg-volvotrucks-us/issues/532 and https://github.com/hlxsites/vg-volvotrucks-us/issues/561. Need searcheng support to complete these tickets. @mollyaljenkins
